app/fastTrack/src/appRegistry/FastTrackBanner.js

```diff
@@ -1,28 +1,62 @@
 // @flow strict
 
+/* eslint-disable import/order */
+
 import * as React from 'react';
-import {
-  SimpleCard,
-  Text,
-  TextButton,
-  StyleSheet,
-} from '@kiwicom/mobile-shared';
+import { AxiosResponse } from 'axios';
+import { View } from 'react-native';
+import { Text, TextButton, StyleSheet, Icon } from '@kiwicom/mobile-shared';
 import { Translation } from '@kiwicom/mobile-localization';
 import { defaultTokens } from '@kiwicom/mobile-orbit';
 
+// Components
 import FastTrackModal from '../components/FastTrackModal';
+import Services from '../components/Services';
+
+// Types
+import type { ThirdPartyAncillariesResponse } from '../types/ThirdPartyAncillariesResponse';
+import type { ThirdPartyAncillariesOrderResponse } from '../types/ThirdPartyAncillariesOrderResponse';
 
 type Props = {|
-  +height: number,
+  +bookingId: number,
+  +requester: (url: string) => Promise<{}>,
 |};
 
 type State = {|
   +isModalVisible: boolean,
+  +documentUrl: string,
 |};
 
 class FastTrackBanner extends React.Component<Props, State> {
   state = {
     isModalVisible: false,
+    documentUrl: '',
+  };
+
+  componentDidMount() {
+    this.fetchFastTrackDocument();
+  }
+
+  fetchFastTrackDocument = async () => {
```

Comment:
Any reason we can not do this in graphql?

Reply:
No GraphQl support for this api.

Reply:
We do all API calls through our GraphQL proxy, so you would need to create your queries and resolvers there.